portal/views.py

```diff
@@ -256,8 +257,11 @@ def dashboard():
         else:
             clusters = ["uutah-prod", "uchicago-prod", "umich-prod"]
 
-        with open("portal/static/news.md", "r") as file:
-            news = file.read()
+        feed = feedparser.parse("https://slateci.io/feed.xml")
```

Comment:
Let's remove the hard-coded URL here and use the `SERVER_NAME` functionality of Flask. I think this can be done via something like:
```python
app.config.get('SERVER_NAME')
```

Reply:
Doing this will allow us to keep the feed in sync with the `portal.conf` file for each environment.